azure-ml-deployment/Python/DeployModel.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Python Deployment Script for Driverless AI Mojo Models
+
+"""
+
+import os
+import shutil 
+import json
+from random import randint
+#Azure ML
+from azureml.core import Workspace
+from azureml.core.model import Model
+from azureml.core.model import InferenceConfig
+from azureml.core.webservice import AciWebservice, AksWebservice, Webservice
+from azureml.core.compute import AksCompute, ComputeTarget
+from azureml.core.webservice import LocalWebservice
+#Azure Blob
+from azure.storage.blob import BlockBlobService, PublicAccess
```

Comment:
just a warning... but this came up in DAI issues with dependencies.... looks like the above module BlockBlobService is now considered legacy.

Reply:
Updated to the latest version of the Python SDK (v12).